object_analytics_node/src/dataset/tr_dataset.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <string>
+#include <vector>
+
+#include "object_analytics_node/dataset/track_dataset.hpp"
```

Comment:
this dataset feature is only for tracker or could be used for others? if only for tracker, I would suggest put .hpp to `object_analytics_node/tracker/track_dataset.hpp`. Save suggests with cpp file to put under tracker folder.

Reply:
> this dataset feature is only for tracker or could be used for others? if only for tracker, I would suggest put .hpp to `object_analytics_node/tracker/track_dataset.hpp`. Save suggests with cpp file to put under tracker folder.

Datasets is a standalone feature which would suppose to be in separated folder, the folder should contain segementation/location dataset later.